technico-v16/app/admin/all-repairs/page.tsx

```diff
@@ -0,0 +1,136 @@
+"use client";
+
+import { useAuth } from "@/app/components/AuthContext";
+import RepairCard from "@/app/components/RepairCard";
+import { useEffect, useState } from "react";
+
+interface Repair {
+    repairId: string;
+    scheduledDate: Date;
+    type: string;
+    description: string;
+    address: string;
+    status: string;
+    cost: number;
+    userId: string;
+    propertyId: string;
+}
+
```

Comment:
- Good use of try catch blocks and handling loading and error state
- Good use of the search functionality
- Loading and error values should be used inside the jsx to provide feedback depending on their state